packages/imperative/src/config/cmd/secure/secure.handler.ts

```diff
@@ -75,6 +75,13 @@ export default class SecureHandler implements ICommandHandler {
             }
         }
 
+        if (params.arguments.prune) {
+            const prunedFiles = config.api.secure.rmUnusedProps();
+            if (prunedFiles.length > 0) {
+                params.response.console.log(`Deleted secure properties for ${prunedFiles.length} missing file(s)`);
```

Comment:
Since this is a destructive operation, I suggest that in addition to displaying the message to the user with the number of files that were deleted, we should also write a log message with all of the detected file names.

If we get a consumer complaint that the wrong stuff was deleted (which might a mistaken recollection of what existed) we will have a record of the files for which we deleted secure properties.